RELEASE-NOTES.adoc

```diff
@@ -19,6 +19,16 @@ endif::[]
 == Table of Contents
 toc::[]
 
+== v0.1.0-alpha.12
+
+=== Breaking changes
+
+* To use NLB functionality we now require EdgeLB built from master (commit https://github.com/mesosphere/dcos-edge-lb/commit/d3547ca23ba9b4a07b91e33562868b731330c855[d3547ca]) or version 1.4.0.
```

Comment:
This should explain what the breaking change is.

Reply:
Done, lmk what you think.